tests/automated_tests/import_example_test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
```

Comment:
The year 2023 shall be set. Looks like header is doubled.

Reply:
That is pre-commit hook issue, what I could do, I could exclude these files